sys-power/nut/nut-2.7.4-r2.ebuild

```diff
@@ -0,0 +1,298 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+inherit autotools bash-completion-r1 fixheadtails user systemd flag-o-matic toolchain-funcs python-single-r1
+
+MY_P=${P/_/-}
+
+DESCRIPTION="Network-UPS Tools"
+HOMEPAGE="https://www.networkupstools.org/"
+# Nut mirrors are presently broken
+SRC_URI="https://networkupstools.org/source/${PV%.*}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~x86-fbsd"
+
+IUSE="cgi gui ipmi snmp +usb selinux ssl tcpd xml zeroconf"
+REQUIRED_USE="gui? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="dev-libs/libltdl:*
```

Comment:
Re `CDEPEND`: 1-letter prefixes for `DEPEND` are reserved for future EAPIs, it would be best to switch to `COMMON_DEPEND` instead.